recipes-kernel/linux-yocto/linux-yocto/lockdep.cfg

```diff
@@ -0,0 +1,99 @@
+CONFIG_EXPERT=y
+CONFIG_KALLSYMS_ALL=y
+# CONFIG_DEBUG_PERF_USE_VMALLOC is not set
```

Comment:
Redone with all settings off dropped (like you pointed out, I don't think we need to force any of them to be off). Boot-tested a resulting kernel and at least the necessary lockdep bits stayed on and were available.